### PR TITLE
Add "conditionally accepted"

### PR DESCRIPTION
**Pull request by b-studios**
I appreciate having our paper listed, but it is not yet fully accepted at OOPSLA and I do not want to spread the impression it is. 
For the time being, maybe we can add "conditionally accepted"?

### CODE REVIEW

**Comment by yallop** [comment]
Thanks, @b-studios!

@dhil: we could avoid this issue in the future by waiting until a paper is unconditionally accepted before adding it to the bibliography. What do you think?

**Comment by b-studios** [comment]
Another option would be to just mark it as TR. Actually, the title of the OOPSLA submission is also slightly different (due to requirements of double blind reviews)

**Comment by dhil** [comment]
> @dhil: we could avoid this issue in the future by waiting until a paper is unconditionally accepted before adding it to the bibliography. What do you think?

Yes!! That's what I have done in the past! This is my mistake, apologies! When it was presented to me I understood it to be (fully) accepted, and I didn't bother to check :-) Lesson learnt.

File: README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ in various programming languages.
   by Ningning Xie and Daan Leijen  
   ([pdf](https://www.microsoft.com/en-us/research/uploads/prod/2020/07/effev.pdf))
 
-* **Effekt: Lightweight Effect Polymorphism for Handlers** (OOPSLA 2020)  
+* **Effekt: Lightweight Effect Polymorphism for Handlers** (conditionally accepted at OOPSLA 2020)
```
Comment (dhil) [inline README.md:127]: I suggest we either delete this entry or change it to "technical report", this is my preferred solution as it complements the software section, then we can update this entry once the paper has been fully accepted at OOPSLA.
```suggestion
* **Effekt: Lightweight Effect Polymorphism for Handlers** (Technical report 2020)  
```

**Comment by b-studios** [comment]
Agreed, technical report sounds good to me. I am very happy that it is already listed since I want people to read it :)

**Comment by dhil** [comment]
Great! Thanks for correcting my error! I will merge as soon as the CI completes successfully.

**Comment by b-studios** [comment]
Great, thanks for adding it in the first place and sorry, if I communicated it wrong somewhere

**Comment by dhil** [comment]
No worries. It was communicated to me by an intermediary. Supposedly, this communication wasn't lossless, or maybe I had instantiated my context with some wrong assumptions ;-))

**Comment by dhil** [comment]
It seems like [bentnib.org](https://bentnib.org) is down at the moment. I will open an issue to notify Bob.

**Comment by b-studios** [comment]
@dhil now that our paper is published (https://dl.acm.org/doi/10.1145/3428194) should we replace the TR with the officially published version? The TR contains additional material, maybe it can be linked in addition to the official paper?

**Comment by dhil** [comment]
Yes!

**Pull request merged**